config/stack/manifests/app.yaml

```diff
@@ -1,5 +1,5 @@
 # Human readable title of application.
-title: Crossplane GCP
+title: Crossplane GCP Provider
 
 overviewShort: The GCP Crossplane provider enables infrastructure management for the Google Cloud Platform.
 overview: |-
```

Comment:
want to make other changes to this file to go from `GCP Crossplane Stack` -> `GCP Crossplane Provider`? there's one below, even though the overviewShort already got updated.

Reply:
Sure thing! Thanks for noticing this :) Will update across other repos as well :+1: